tests/lib/nested.sh

```diff
@@ -731,6 +736,8 @@ nested_start_core_vm_unit() {
 
     # ensure we have a log dir
     mkdir -p "$NESTED_LOGS_DIR"
+    # make sure we start with clean log file
+    echo > "${NESTED_LOGS_DIR}/serial.log"
```

Comment:
mmm this may conflict with the changes @sergiocazzolato proposed in https://github.com/snapcore/snapd/pull/9348